docs/Conformance.md

```diff
@@ -23,12 +23,12 @@ Search parameters defined in the specification can be found by browsing the R4 F
 In addition, the following search parameters are supported on all resources:
 * `_id`
 * `_lastUpdated`
+* `_tag`
 * `_profile`
 * `_security`
 * `_source`
-* `_tag`
```

Comment:
not sure why move up _tag, the old list is and ordered list.

Reply:
I reordered it to match https://www.hl7.org/fhir/search.html